README.md

```diff
@@ -232,14 +199,11 @@ There is a better way:
 ```ruby
 require 'hanami/model'
 
-class ArticleRepository
-  include Hanami::Repository
-
-  def self.most_recent_by_author(author, limit = 8)
-    query do
-      where(author_id: author.id).
-        order(:published_at)
-    end.limit(limit)
+class ArticleRepository < Hanami::Repository
```

Comment:
@AMHOL Thanks for the explanation. But this would be an exception in the entire framework where we always have mixins to include.